web/src/containers/ApdExport.js

```diff
@@ -0,0 +1,82 @@
+import { Button } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Waypoint from './ConnectedWaypoint';
+import { printApd } from '../actions/print';
+import { Section } from '../components/Section';
+import { FileDownload } from '../components/Icons';
+
+const Bold = ({ children }) => (
+  <span className="ds-u-font-weight--bold">{children}</span>
+);
+Bold.propTypes = { children: PropTypes.node.isRequired };
+
+const Italic = ({ children }) => (
+  <span className="ds-u-font-style--italic">{children}</span>
+);
+Italic.propTypes = { children: PropTypes.node.isRequired };
+
+const ExportAndSubmit = ({ printApd: print }) => (
+  <Waypoint id="export-and-submit">
+    <Section isNumbered id="export-and-submit" resource="exportAndSubmit">
+      <p>
```

Comment:
@jameshupp I put the content in the code instead of the YAML because the YAML's Markdown converter thing does lists really poorly.  In order to get proper numbered lists, I needed to put the text here directly.

Reply:
What converter are we using, and would it make sense to consider adjusting it?

Reply:
We're using [markdown-it](https://www.npmjs.com/package/markdown-it) and I'm totally open to switching to something else.